site/docs/v1/tech/apis/scim/_scim-response-codes.adoc

```diff
@@ -14,21 +14,25 @@ ifdef::delete[]
 endif::[]
 
 |400
-|The request was invalid and/or malformed. The response will contain an link:/docs/v1/tech/apis/scim[SCIM Errors] JSON Object with the specific errors. This status will also be returned if a paid FusionAuth license is required and is not present.
+|The request was invalid and/or malformed. The response will contain a link:/docs/v1/tech/apis/scim[SCIM Error] JSON Object with the specific errors. This status will also be returned if a paid FusionAuth license is required and is not present.
 
+ifndef::unauthenticated[]
 |401
 |You did not supply a valid JWT in your Authorization header. The response will be empty. Ensure you've correctly set up Entities and performed a link:/docs/v1/tech/apis/authentication#client-credentials[Client Credentials] grant.
+endif::[]
 
-ifndef::create[]
+ifndef::no_missing[]
 |404
 |The object doesn't exist. The response will be empty.
 endif::[]
 
 |500
 |There was an internal error. A stack trace is provided and logged in the FusionAuth log files. The response will be empty.
 
+ifndef::no_search[]
 |503
 |The search index is not available or encountered an exception so the request cannot be completed. The response will contain a JSON body.
+endif::[]
 
 ifndef::no_webhook_event[]
```

Comment:
The inverse seems to be more common: `webhook_event` as illustrated here: `site/docs/v1/tech/apis/_standard-post-response-codes.adoc`

Reply:
In the case of SCIM reponse codes. webhooks are much more common because almost all the endpoints are CRUD endpoints for Users or Group Members.

Reply:
Sounds good.